src/main/java/com/testlio/plugins/jenkins/enums/AppTypeEnum.java

```diff
@@ -1,18 +1,19 @@
 package com.testlio.plugins.jenkins.enums;
 
+import lombok.Getter;
+
+@Getter
 public enum AppTypeEnum {
-    DEVICE_APP("Mobile App"),
-    DEVICE_WEB("Mobile Web"),
-    BROWSER("Desktop Web");
+    DEVICE_APP("Mobile App","OTHER"),
```

Comment:
I think elsewhere the enum value for this has been `ANDROID` or `IOS` for "Mobile App". Not sure if it's relevant, but mentioning just in case

Reply:
The other enum is being used to check Device app specific type